backend/src/main/java/capstone/backend/security/service/UserAuthUtils.java

```diff
@@ -5,12 +5,8 @@
 
 @Component
 public class UserAuthUtils {
-    //TODO implement better password validation
     public void validatePassword(String password) throws InvalidCredentialsException {
         if(password.length() < 3) throw new InvalidCredentialsException("Password too short");
```

Comment:
Better do the password validation in frontend, dont use ressources from backend when you could validate the data before.